portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,24 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function getMessage() {
```

Comment:
Are these 3 functions needed? I think getMessageArrowFunctions does everything you need, so you don't need to include the others.